src/worker/storage.ts

```diff
@@ -71,4 +90,19 @@ export default class Storage {
   fetchAllFiles(): Promise<StoredWordList[]> {
     return this.db.files.toArray();
   }
+  /**
+   * Update bcp47 tag to database
+   * @param bcp47Tag
+   */
+  updateBCP47Tag(bcp47Tag: string): Promise<void> {
+    return this.db.transaction("readwrite", this.db.packageInfo, async () => {
+      await this.db.packageInfo.put({ bcp47Tag, id: 0 });
```

Comment:
Could you please extact `0` as a constant?

```typescript
/**
 * The key of the ONLY StoredPackageInfo record.
 */
const PACKAGE_ID = 0;
```

And change this line to:
```suggestion
      await this.db.packageInfo.put({ bcp47Tag, id: PACKAGE_ID });
```